models/contacts.js

```diff
@@ -1,19 +1,90 @@
-// const fs = require('fs/promises')
+const fs = require("fs/promises");
+const path = require("path");
+const contactsPath = path.resolve("./models/contacts.json");
 
-const listContacts = async () => {}
+const listContacts = async () => {
+  try {
+    const readContacts = await fs.readFile(contactsPath);
+    return JSON.parse(readContacts);
+  } catch (error) {
+    console.log("Load error", error.message);
+  }
+};
 
-const getContactById = async (contactId) => {}
+const getContactById = async (contactId) => {
+  try {
+    const contacts = await listContacts();
+    const getContact = contacts.find((contact) => contact.id === contactId);
+    return getContact;
+  } catch (error) {
+    console.log("Get error", error.message);
+  }
+};
 
-const removeContact = async (contactId) => {}
+const removeContact = async (contactId) => {
+  try {
+    const contacts = await listContacts();
+    const getContact = await getContactById(contactId);
+    const removeContact = contacts.filter((cont) => cont.id !== contactId);
```

Comment:
trzymaj jedna konwencje swojego kodu. Jesli poprzednio zrobilas `contact` callbacku, to tutaj tez zrob `contact`, a nie `cont`.